evals/utils/api_utils.py

```diff
@@ -1,11 +1,16 @@
 """
 This file defines various helper functions for interacting with the OpenAI API.
 """
+import concurrent
 import logging
+import os
 
 import backoff
 import openai
 
+EVALS_THREAD_TIMEOUT = float(os.environ.get("EVALS_THREAD_TIMEOUT", "40"))
```

Comment:
nit: maybe `EVALS_COMPLETION_TIMEOUT`?

Reply:
We had `EVALS_THREAD_TIMEOUT` historically and I think some people actually use this env var so I dont want to change it on them